indico/modules/events/editing/controllers/frontend.py

```diff
@@ -37,6 +38,12 @@ def _process(self):
         return WPEditingView.render_template(
             'editing.html',
             self.event,
-            editable=self.editable,
-            contribution=self.contrib
+        )
+
+
+class RHEditableTypeList(RHEditingBase):
```

Comment:
We need a permission check in this RH. `RHEditablesList` (the underlying API) probably needs to be updated as well since it currently requires management access. But editors who aren't managers can still use the new view, so the API must let them access it as well.